webapp/layouts/default.vue

```diff
@@ -4,30 +4,30 @@
       <ds-container class="main-navigation-container" style="padding: 10px 10px;">
         <div>
           <ds-flex class="main-navigation-flex">
-            <ds-flex-item :width="{ lg: '3.5%' }" />
-            <ds-flex-item :width="{ base: '80%', sm: '80%', md: '80%', lg: '15%' }">
+            <ds-flex-item :width="{ base: '142px' }">
               <nuxt-link :to="{ name: 'index' }" v-scroll-to="'.main-navigation'">
                 <ds-logo />
               </nuxt-link>
             </ds-flex-item>
             <ds-flex-item
-              :width="{ base: '20%', sm: '20%', md: '20%', lg: '0%' }"
+              :width="{ base: '40%', sm: '40%', md: '40%', lg: '0%' }"
               class="mobile-hamburger-menu"
             >
-              <ds-button icon="bars" @click="toggleMobileMenuView" right />
+              <base-button icon="bars" @click="toggleMobileMenuView" circle />
             </ds-flex-item>
             <ds-flex-item
-              :width="{ base: '85%', sm: '85%', md: '50%', lg: '50%' }"
+              :width="{ base: '45%', sm: '45%', md: '45%', lg: '50%' }"
               :class="{ 'hide-mobile-menu': !toggleMobileMenu }"
+              style="flex-shrink: 0; flex-grow: 1;"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-15T16:33:55Z" title="Wednesday, January 15th 2020, 5:33:55 pm +01:00">Jan 15, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -4,30 +4,30 @@
       <ds-container class="main-navigation-container" style="padding: 10px 10px;">
         <div>
           <ds-flex class="main-navigation-flex">
-            <ds-flex-item :width="{ lg: '3.5%' }" />
-            <ds-flex-item :width="{ base: '80%', sm: '80%', md: '80%', lg: '15%' }">
+            <ds-flex-item :width="{ base: '142px' }">
               <nuxt-link :to="{ name: 'index' }" v-scroll-to="'.main-navigation'">
                 <ds-logo />
               </nuxt-link>
             </ds-flex-item>
             <ds-flex-item
-              :width="{ base: '20%', sm: '20%', md: '20%', lg: '0%' }"
+              :width="{ base: '40%', sm: '40%', md: '40%', lg: '0%' }"
               class="mobile-hamburger-menu"
             >
-              <ds-button icon="bars" @click="toggleMobileMenuView" right />
+              <base-button icon="bars" @click="toggleMobileMenuView" circle />
             </ds-flex-item>
             <ds-flex-item
-              :width="{ base: '85%', sm: '85%', md: '50%', lg: '50%' }"
+              :width="{ base: '45%', sm: '45%', md: '45%', lg: '50%' }"
               :class="{ 'hide-mobile-menu': !toggleMobileMenu }"
+              style="flex-shrink: 0; flex-grow: 1;"
```


why did you use inline styling in these cases? something to do with the styleguide?
is this to avoid targeting the class and applying `!important`... just guessing...

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-22T07:49:37Z" title="Wednesday, January 22nd 2020, 8:49:37 am +01:00">Jan 22, 2020</time>_

---


It's to override inline styles applied by the styleguide – and I thought it's harder to miss this way than with applying a class and `!important` styles. Needs to be refactored either way!